db/db.go

```diff
@@ -11,5 +11,5 @@ type DB interface {
 	DeleteItem(string, string) error
 	GetItem(listID string, itemID string) (*data.Item, error)
 	GetItemsOnList(string) (*[]data.Item, error)
-	UpdateItem(string, string, string) (*data.Item, error)
+	UpdateItem(string, string, string, *bool) (*data.Item, error)
```

Comment:
UpdateItem takes an extra param of whether the item should be changed to be complete or incomplete, if it's nil then it won't be changed, if name is an empty string it won't be changed